voucher/urls.py

```diff
@@ -3,8 +3,11 @@
 
 # SharedAPIRootRouter is automatically imported in global urls config
 router = SharedAPIRootRouter()
-router.register(r'voucher/wallets', WalletViewSet, base_name='voucher_wallets')
+router.register(r'voucher/wallets/voucher', VoucherWalletViewSet, base_name='voucher_wallets')
+router.register(r'voucher/wallets/coffee', CoffeeWalletViewSet, base_name='coffee_wallets')
+router.register(r'voucher/registerlogs/voucher', WorkLogViewSet)
+router.register(r'voucher/registerlogs/coffee', RegisterLogViewSet)
+router.register(r'voucher/uselogs/voucher', VoucherUseLogViewSet)
+router.register(r'voucher/uselogs/coffee', CoffeeUseLogViewSet)
```

Comment:
Angående API-endepunktene:
Jeg syntes at man burde rename de slik at man kan se forskjell på kaffe og bonger. Er ikke et stort problem for meg å gjøre om på frontenden heller.